### PR TITLE
move 'what is arroyo for' docs to public docs

### PR DESCRIPTION
**Pull request by volokluev**
Moving this doc into here

https://www.notion.so/sentry/What-is-Arroyo-for-a9bca6856abe4c8a90338783b252e581

### CODE REVIEW

**Review by nikhars — APPROVED** [review]

**Comment by nikhars** [comment]
I approved. But maybe Lyn might want to wait for @lynnagara approval

**Review by lynnagara — APPROVED** [review]
Thanks

**Pull request merged**
